packages/event-emitter/scripts/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+#
+# Build script for `@react-native/event-emitter`. Run using `npm run build`.
+#
+
+set -e
+set -u
+
+ROOT_DIR=$(dirname "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)")
+
+babel "$ROOT_DIR/src/EventEmitter.js" --out-dir "$ROOT_DIR/lib"
+prettier "$ROOT_DIR/lib/EventEmitter.js" --write
+
+cp "$ROOT_DIR/src/EventEmitter.js" "$ROOT_DIR/lib/EventEmitter.js.flow"
+cp "$ROOT_DIR/src/EventEmitter.d.ts" "$ROOT_DIR/lib/EventEmitter.d.ts"
```

Comment:
A potentially interesting choice that I've made here is checking in the build artifacts at `packages/event-emitter/lib`.

The reason I am doing this is because in order for `@react-native/event-emitter` to be resolvable locally using Yarn Workspaces without requiring that developers independently run `cd packages/event-emitter; yarn` (which is cumbersome), we can instead check these built artifacts in so that rebasing onto the main branch simply gets you the newly built files. That is, the onus of updating these files is on whoever changes (and presumably tests) the source files.

As a consequence, one strange thing I'm doing here is running `prettier` on the Babel output file. This is so that `yarn run format-check` does not fail on that build artifact in CI. 🤷‍♂️

Reply:
> A potentially interesting choice that I've made here is checking in the build artifacts at packages/event-emitter/lib.

FYI That's consistent with what we do for `react-native-codegen`